packages/twenty-front/.eslintrc.cjs

```diff
@@ -41,6 +41,7 @@ module.exports = {
         ],
       },
     ],
+    'no-extra-boolean-cast': 'off',
```

Comment:
I disabled it because the new workspace rule: `explicit-boolean-predicates-in-if` kind of conflicts with this one.

This new `explicit-boolean-predicates-in-if` rule enforces that `if` statements use a boolean condition, so if we want to use `if (myNonBooleanVariable)`, we can't because it is not a boolean.
But `no-extra-boolean-cast` removes boolean casts deemed redundant, so we cannot use `if (!!myNonBooleanVariable)` or `if (Boolean(myNonBooleanVariable))` either.

Not sure if this is intentional. I guess the intent of the new rule is to enforce using type guards like `if (isDefined(myNonBooleanVariable))`, but it seems strange to me that we can write this:
- `myNonBooleanVariable && <MyComponent />`
- `if (!myNonBooleanVariable) { ... }`
But not this:
- `if (myNonBooleanVariable) return <MyComponent />`
- `if (!!myNonBooleanVariable) { ... }`

I reactivated the rule as it has nothing to do with this PR, but might be worth discussing about this with the team :)